bin/search.py

```diff
@@ -27,12 +25,12 @@ def init_bench(query):
     with db.cursor(connection) as cursor:
         return api.search_from_text_query(cursor, query)
 
-
 if __name__ == "__main__":
     """Execute the sql search function with the query passed as an argument. 
     How to execute: python3 -m search.py query"""
     query = " ".join(sys.argv[1:])  
     logging.debug(f"Query is: {query}")
-    init_bench(query)
-    with open(OUTFILE, "r") as result_file:
-        print(pd.read_json(result_file, lines=True))
+    results = init_bench(query)
+    query_file_name = query.replace (" ", "-") + ".json"
```

Comment:
https://nadeauinnovations.com/post/2020/11/python-tricks-replace-all-non-alphanumeric-characters-in-a-string/

Reply:
keeping the spaces as it is made for finesse-frontend